ncm-hostsaccess/src/main/perl/hostsaccess.pod

```diff
@@ -2,42 +2,49 @@
 # ${developer-info}
 # ${author-info}
 
-
 =head1 NAME
 
-hostsaccess: NCM component to control /etc/hosts.allow and hosts.deny files
+hostsaccess: NCM component to control C<< /etc/hosts.allow >> and C<< hosts.deny >> files.
 
 =head1 DESCRIPTION
 
-The I<hostsaccess> component manages the configuration files /etc/hosts.allow
-and /etc/hosts.deny.  Few checks are done on the given configuration to 
-allow all of the supported wildcarding. 
+The I<hostsaccess> component manages the configuration files C<< /etc/hosts.allow >>
+and C<< /etc/hosts.deny >>. Few checks are done on the given configuration to
+allow all of the supported wildcarding.
 
 =head1 RESOURCES
 
-=head2 /software/components/hostsaccess/allow
+=over
+
+=item * C<< /software/components/hostsaccess/allow >>
 
 A list where each entry consists of a named list with the keys: daemon
 and host.  Both of the keys take strings as values and hence can
 support the full wildcarding syntax.  These entries are allowed to
-access the daemon. 
+access the daemon.
 
-NOTE: The daemon name MUST be encoded with the pan escape()
-function. This allows daemon lists to be used in the specification. 
+B<NOTE:> The daemon name MUST be encoded with the pan escape()
+function. This allows daemon lists to be used in the specification.
 
-=head2 /software/components/hostsaccess/deny
+=item * C<< /software/components/hostsaccess/deny >>
 
 A list where each entry consists of a named list with the keys: daemon
 and host.  Both of the keys take strings as values and hence can
 support the full wildcarding syntax.  These entries are denied access
-to the daemon. 
+to the daemon.
+
+B<NOTE:> The daemon name MUST be encoded with the pan escape()
+function. This allows daemon lists to be used in the specification.
 
-NOTE: The daemon name MUST be encoded with the pan escape()
-function. This allows daemon lists to be used in the specification. 
+=back
 
 =head1 EXAMPLE
 
-"/software/components/hostsaccess/allow" = push(nlist("daemon","slapd",
-                                                 "host","127.0.0.1"));
+  "/software/components/hostsaccess/allow" = push(
```

Comment:
This does not follow the NOTE from line 36 as the daemon name is not actually escaped. Also s/push/append/ please.